src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
Should these be in the PublicAPI.Shipped.txt, since we have effectively already shipped these?